_episodes/06-ignore.md

```diff
@@ -48,7 +48,7 @@ What's worse,
 having them all listed could distract us from changes that actually matter,
 so let's tell Git to ignore them.
 
-We do this by creating a file in the root directory of our project called `.gitignore`:
+We do this by creating a file in the root directory of our project called `.gitignore` specifying we want to ignore the `.dat` files and `results` directory we just created.
```

Comment:
I think this added text here is repetitive. The line just below the shell the output of `cat` says what is being ignored, and the line just above this one states our intent to ignore files that aren't going to matter to our project. Would you mind changing this text back to the original wording?

Reply:
Alternatively, this sentence could be modified to further state how we plan to change the .gitignore file. e.g. 

> We do this by creating a file in the root directory of our project called `.gitignore` and adding lines specifying which things we'd like to ignore in our project.